source/renderer/app/stores/AppStore.js

```diff
@@ -25,6 +26,9 @@ export default class AppStore extends Store {
   @observable gpuStatus: ?GpuStatus = null;
   @observable numberOfEpochsConsolidated: number = 0;
   @observable previousRoute: string = ROUTES.ROOT;
+  @observable availableAppVersion: ?string = null;
```

Comment:
@tomothespian I feel like these app version related observables and actions should be moved into the `NodeUpdateStore`. Can you please move them?